### PR TITLE
Update issue template with discord link

### PR DESCRIPTION
**Pull request by pleku**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,16 +1,16 @@
 <!--
-Please read these instructions through before submitting an issue, Thank You!
+Please READ these instructions & USE the issue template below, Thank You!
 
 Only use GitHub issues for bugs and feature requests.   
-For general support from the community, use https://vaadin.com/forum or https://gitter.im/vaadin/flow instead.
+For general support from the community, use https://vaadin.com/forum or Vaadin discord chat https://discord.gg/MYFq5RTbBn instead.
 
-NOTE: Issues concerning certain component should go to the corresponding components repository like [vaadin/vaadin-grid-flow](https://github.com/vaadin/vaadin-grid-flow).
+NOTE: Issues concerning certain UI component should go to the components repository https://github.com/vaadin/vaadin-flow-components .
```
Comment (caalador) [inline .github/ISSUE_TEMPLATE.md:7]:
```suggestion
NOTE: Issues concerning certain UI components should go to the components repository https://github.com/vaadin/vaadin-flow-components .
```

Comment (caalador) [reply .github/ISSUE_TEMPLATE.md:7]: Also should we update certain to more point to components found at https://vaadin.com/components somehow

**Review by caalador — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (pleku) [reply .github/ISSUE_TEMPLATE.md]: Yes probably, but we need to also take more steps for updating this to the "new way" https://docs.github.com/en/free-pro-team@latest/github/building-a-strong-community/about-issue-and-pull-request-templates

Comment (pleku) [reply .github/ISSUE_TEMPLATE.md]: So this is probably enough now

**Review by caalador — APPROVED** [review]

**Comment by pleku** [comment]
Does this need to be cherry picked ? I would expect it to just use the on in the master

**Comment by caalador** [comment]
I'm like 50/20 on is as 2.4, 2.5, 4.0 and 5.0 are live branches and reading the README in those would then lead to gitter andcomponent repo. I'm not against not picking them, just felt that live branches should probably have the needed information.

**Comment by pleku** [comment]
Not sure what you mean - this is the issue template, not the readme ?

**Pull request merged**